src/Symfony/Component/ErrorHandler/ErrorRenderer/ErrorRenderer.php

```diff
@@ -62,7 +62,7 @@ public function render($exception, string $format = 'html'): string
             throw new ErrorRendererNotFoundException(sprintf('No error renderer found for format "%s".', $format));
         }
 
-        if (!$exception instanceof FlattenException) {
+        if ($exception instanceof \Exception) {
```

Comment:
Also makes mere mortal like me reason about the code more easily as `create()` takes an `\Exception` instance.